### PR TITLE
Create ru.po

### PR DESCRIPTION
**Pull request by vorons**

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -0,0 +1,662 @@
...
+#: data/com.github.finefindus.eyedropper.desktop.in.in:5
+#: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:8
+#: src/widgets/about_window.rs:74
+msgid "A powerful color picker and formatter"
+msgstr "Мощный инструмент для выбора цвета и средство форматирования"
+
+#. Translators: Search terms to find this application. Do NOT translate or localize the semicolons! The list MUST also end with a semicolon!
+#: data/com.github.finefindus.eyedropper.desktop.in.in:11
+msgid "Gnome;GTK;Color;Color Picker;Picker;Palette;"
+msgstr ""
```
Comment (FineFindus) [inline po/ru.po:41]: Please translate them as well, they are used in the overview search and are language specific.

File: po/ru.po [hunk 2]
```diff
@@ -0,0 +1,662 @@
...
+msgid "F12 (Ultralume 30, Philips TL83)"
+msgstr "F12 (Ультралюм 30, Philips TL83)"
+
+#: data/resources/ui/preferences.ui:125
+msgid "CIE Standard Observer"
+msgstr ""
+
+#: data/resources/ui/preferences.ui:126
+msgid "The CIE standard observer's field of view"
+msgstr ""
```
Comment (FineFindus) [inline po/ru.po:392]: I think you might have forgotten to translate these?

**Review by FineFindus — CHANGES_REQUESTED** [review]
Thanks for the translation!
I found two minor issues which should addressed before merging, and please also add the language code to `po/LINGUAS`.

Will you continue to maintain this translation?

**Comment by vorons** [comment]
Several corrections added.

I can support the translation as far as possible.

**Comment by FineFindus** [comment]
Thanks!

**Pull request merged**